tasks/species_typing/task_kaptive.wdl

```diff
@@ -0,0 +1,152 @@
+version 1.0
+
+task kaptive {
+  # Inputs
+  input {
+    File assembly
+    String samplename
+    String kaptive_docker_image = "quay.io/staphb/kaptive:2.0.3"
+    # Parameters
+    Int start_end_margin = 10 # determines flexibility in identifying the start and end of a locus - if this value is 10, a locus match that is missing the first 8 base pairs will still count as capturing the start of the locus (default: 10) 
+    Float min_identity = 90.0 # minimum required percent coverage for the gene BLAST search via tBLASTn (default: 90.0)
+    Float min_coverage = 80.0 # minimum required percent identity for the gene BLAST search via tBLASTn (default: 80.0)
+    Float low_gene_id = 95.0 # percent identity threshold for what counts as a low identity match in the gene BLAST search (default: 95.0)
+    #Int min_assembly_piece = 100 # smallest piece of the assembly (measured in bases) that will be included in the output FASTA files (default: 100)
+    #Int gap_fill_size = 100 # size of assembly gaps to be filled in when producing the output FASTA files
+  }
+
+  command <<<
+    #find absolute path of kaptive directory
+    KAPTIVE_DIR=$(dirname "$(which kaptive.py)")
+    # capture date and version
+    # Print and save date
+    date | tee DATE
+    # Print and save version
+    kaptive.py --version | tee VERSION 
+    # Run Kaptive on the input assembly with the --all flag and output with samplename prefix
+    kaptive.py \
+    ~{'--start_end_margin ' + start_end_margin} \
+    ~{'--min_gene_id ' + min_identity} \
+    ~{'--min_gene_cov ' + min_coverage} \
+    ~{'--low_gene_id ' + low_gene_id} \
```

Comment:
Any particular reason why `--low_gene_id` and `--start_end_margin` are used for K locus querying and not below for the OC locus?

Reply:
There is a reason, and that reason is that I forgot to put them there 😅. I'll add them in now. 

I do have all those values set to the defaults currently, so there shouldn't be any change in behavior.